### DISCUSSION TITLE
Goodbye

### DISCUSSION THREAD

**Discussion by marcosoliveira20** [maiconspa/codando-discordbot#62] [Ideas]
Como possuímos o endereço de e-mail da mesma talvez fosse interessante fazer um serviço de disparo de e-mail, dando um adeus para ela. 

**Assim que o user sai do servidor o bot não tem mais autorização para mandar mensagem. O discord lança uma exception caso a gente tente fazer isso**

**Comment by maiconspa** [comment]
Não coletamos mais o e-mail da pessoa, temos que pensar em uma alternativa ou fechar a issue
